NEWS.md

```diff
@@ -1,3 +1,15 @@
+## 2.2.0 - Unreleased
+
+## 2.1.0 - Released
+
+This release includes tuning environment settings
+
+[Full Changelog](https://github.com/folio-org/edge-oai-pmh/compare/v2.0.1...v2.1.0)
+
+### Stories
```

Comment:
There is no story for it but are we capturing this change https://github.com/folio-org/edge-oai-pmh/pull/24 ?

Reply:
There was only one change in readme.md file

Reply:
Maybe it's fine then.